ebpf/event_common.go

```diff
@@ -93,6 +93,22 @@ type ConnectionStats struct {
 	Type      ConnectionType      `json:"type"`
 	Family    ConnectionFamily    `json:"family"`
 	Direction ConnectionDirection `json:"direction"`
+	Conntrack *Conntrack          `json:"conntrack"`
+}
+
+// TODO: add to String()
+//easyjson:json
+type Conntrack struct {
+	OrigSrcIP string `json:"orig_src_ip"`
```

Comment:
Do we actually need to send back `Orig`? It should just be whatever we see in our source IP and dest IP, no?

Reply:
Good point. If should always match the laddr.